cubi_tk/archive/common.py

```diff
@@ -117,6 +118,19 @@ def traverse_project_files(directory, followlinks=True):
             yield get_file_attributes(os.path.join(path, filename), root)
 
 
+def load_variables(template_dir):
```

Comment:
I simplified this function from the `isa_tpl` version. Just to check, is the `extra` argument used for anything in the `archive`?

Reply:
Not in use, I don't see any situation which would require programmatically updating the configuration.